ourgroceries/__init__.py

```diff
@@ -133,6 +137,16 @@ async def create_list(self, name, list_type='SHOPPING'):
         }
         return await self._post(ACTION_LIST_CREATE, other_payload)
 
+    async def create_category(self, name):
+        """Create a new category."""
+        _LOGGER.debug('ourgroceries create_list')
```

Comment:
change `_LOGGER.debug('ourgroceries create_list')` to `_LOGGER.debug('ourgroceries create_category')`